plugins/inputs/mesos/README.md

```diff
@@ -142,6 +143,44 @@ Mesos master metric groups
     - master/tasks_staging
     - master/tasks_starting
 
+- operations
+    - master/operations/total
+		- master/operations/pending
```

Comment:
I think that the suffixes here (`total`, `pending`, `recovering`, `unreachable`) don't match the ones that Mesos exposes.

[Here](https://github.com/apache/mesos/blob/e5f48794f0354f8179f4302b308fb89a9f7caf7c/src/master/metrics.cpp#L606-L614) is the code that generates the metrics, and [here's](https://mesos.apache.org/documentation/latest/monitoring/#operations) the documentation describing which metrics are counters and which are gauges.

The gist is that Mesos uses gauges for the non-terminal states (`pending`, `recovering`, or `unreachable`) and counters for the terminal states (`finished`, `error`, `dropped`, or `gone_by_operator`).

There are also three more metrics exposed by the master: `master/valid_operation_status_updates`, `master/invalid_operation_status_updates`, and `master/messages_operation_status_update`.

Reply:
Thanks! I addressed your comments.